index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles.css">
+    <title>Title</title>
+</head>
+<body>
+
+<header>
+
+    <nav class="navigation">
+        <ul>
+            <li><a class="nlink" href="#projects-sec">Case Studies</a></li>
+            <li><a class="nlink" href="#services-sec">Services</a></li>
+            <li><a class="nlink" href="#end-sec">Contact Us</a></li>
+        </ul>
+    </nav>
+
+    <div class="center-title">
+        <h1> Your website is an investment in the success of your business.</h1>
+            <p class="second-title">We are here to help you succeed.</p>
+        <button class="button" type="button">Let's work together</button>
+    </div>
+    <figure><img src="assets/squiggle_line_stripe.svg" height="50" width="50px" alt="squiggle_line_stripe"></figure>
+</header>
+
+
+<main>
+
+    <section id="intro-sec">
+        <div class="introduction">
+        <h2>Rekupe is a digital agency in Los Angeles.</h2>
+            <br />
+            <h2>We use strategic UX, SEO optimization and beautiful visual designs to help your business grow.</h2>
+            <br />
+            <h2>We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.</h2>
+            <p class="intro-quest">Have a question? <a class="click-link" href="">Click here</a> here to contact us!</p>
+    </div>
+    </section>
+
+
+    <section id="projects-sec">
+        <article class="article1">
+        <div class="container1"><h2 class="title1">Rolling Stone</h2>
+            <img src="assets/squiggle_line.svg" height="50" width="50px" alt="squiggle_line">
+            <p class="p1">You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+                <a class="view-link">view project -</a></div>
+            <span class="image-wrap"></span>
+            <img src="assets/work-image-01.jpeg" alt="image1">
```

Comment:
De plaatje schalen nog niet echt mee, wellicht helpt het als de span ook daadwerkelijk om de afbeelding heen staat
```suggestion
            <span class="image-wrap">
                <img src="assets/work-image-01.jpeg" alt="image1">
            </span>
```